<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy flash S4 </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Bp_flash">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug console</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 1 :  Select Rom location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_filelocation">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug box</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kies in use</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Root password :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>115</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug Kext</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kext Kies loaded</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Device connected</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Hash MD5 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hex_lineedit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 2 : Flash</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Device flash with success</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="2">
      <widget class="QCheckBox" name="SYSTEM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>SYSTEM</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="RPM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>RPM</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="CACHE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CACHE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="APNHLOS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>APNHLOS</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="MDM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>MDM</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="bpFile_Recovery">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_RECOVERY</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="TZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>TZ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="bpFile_ABOOT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_ABOOT</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="ABOOT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ABOOT</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="RECOVERY">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>RECOVERY</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QCheckBox" name="SBL2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>SBL2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="bpFIle_CACHE">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_Cache</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="bpFile_MDM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FIle_MDM</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="bpFile_System">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_System</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="bpFile_HIDDEN">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_HIDDEN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="bpFile_TZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_TZ</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="bpFile_APNHLOS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_APNHLOS</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="HIDDEN">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>HIDDEN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="SBL3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>SBL3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="BOOT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>BOOT</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="bpFile_SLB2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FILE_SBL2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bpFile_SLB3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>File_SLB3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="bpFile_BOOT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FILE_BOOT</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="bpFIle_RPM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FIle_RPM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>EXPERT MODE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
